opslevel/resource_opslevel_domain_test.go

```diff
@@ -0,0 +1,61 @@
+package opslevel
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccExampleResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
```

Comment:
does this mean all our tests are now going to reachout to an OpsLevel account?

I was hoping we could have mostly unit test with only a few integration tests.  Is this not possible?

Reply:
I'm pretty sure we can test the majority (all?) of this code locally without any accounts. I just haven't gotten back to this part of the code yet.

Reply:
Scratch that. It's all local

Reply:
Update on this, it _can_ be local but primary testing in Terraform requires some interaction with actual infrastructure. We can specify testing that is "plan" only, like testing `terraform plan`. By default testing goes through a `terraform apply` workflow